Day-01/haskell/marie/Day-01.hs

```diff
@@ -0,0 +1,11 @@
+import Data.List (sort, transpose)
+
+main = do
+  input <- getContents
```

Comment:
It might be preferable if this read the input from a file, like all the other solutions? At the very least, it should probably prompt the user to enter the input

Reply:
i expect the user to pipe a file into stdin with their shell.
if it reads from a file the user would have to look at the code anyway for the filename, so it doesn't make a big difference imo

Reply:
This was not obvious to me, but I'm also not really familiar with Haskell. Maybe a comment would be appropriate?